node/pkg/watchers/evm/chain_config.go

```diff
@@ -58,13 +62,13 @@ var (
 		// Polygon supports polling for finalized but not safe: https://forum.polygon.technology/t/optimizing-decentralized-apps-ux-with-milestones-a-significantly-accelerated-finality-solution/13154
 		vaa.ChainIDPolygon: {Finalized: true, Safe: false, EvmChainID: 137, PublicRPC: "https://polygon-bor-rpc.publicnode.com"},
 
-		vaa.ChainIDAvalanche: {Finalized: false, Safe: false, EvmChainID: 43114, PublicRPC: "https://avalanche-c-chain-rpc.publicnode.com"},
-		vaa.ChainIDOasis:     {Finalized: false, Safe: false, EvmChainID: 42262, PublicRPC: "https://emerald.oasis.dev/"},
+		vaa.ChainIDAvalanche: {InstantFinality: true, Finalized: true, Safe: true, EvmChainID: 43114, PublicRPC: "https://avalanche-c-chain-rpc.publicnode.com"},
+		vaa.ChainIDOasis:     {InstantFinality: true, Finalized: false, Safe: false, EvmChainID: 42262, PublicRPC: "https://emerald.oasis.dev/"},
 		// vaa.ChainIDAurora:    Not supported in the guardian.
-		vaa.ChainIDFantom:   {Finalized: false, Safe: false, EvmChainID: 250, PublicRPC: "https://fantom-rpc.publicnode.com"},
+		vaa.ChainIDFantom:   {InstantFinality: true, Finalized: false, Safe: false, EvmChainID: 250, PublicRPC: "https://fantom-rpc.publicnode.com"},
 		vaa.ChainIDKarura:   {Finalized: true, Safe: true, EvmChainID: 686, PublicRPC: "https://eth-rpc-karura.aca-api.network/"},
 		vaa.ChainIDAcala:    {Finalized: true, Safe: true, EvmChainID: 787, PublicRPC: "https://eth-rpc-acala.aca-api.network/"},
-		vaa.ChainIDKlaytn:   {Finalized: false, Safe: false, EvmChainID: 8217}, // As of Feb 2025, can't find a working public node.
+		vaa.ChainIDKlaytn:   {InstantFinality: true, Finalized: true, Safe: true, EvmChainID: 8217}, // As of Feb 2025, can't find a working public node.
```

Comment:
Klaytn is now Kaia https://docs.kaia.io/misc/kaia-transition/faq-chain-transition/

It does **not** support `finalized` or `safe`. Tested using `https://public-en.node.kaia.io` which is documented here: https://docs.kaia.io/references/public-en/